src/Sentry.Unity.Tests/Sentry.Unity.Tests.csproj

```diff
@@ -8,6 +8,14 @@
         <HintPath>$(UnityPath)\UnityEngine.dll</HintPath>
         <Private>false</Private>
       </Reference>
+      <Reference Include="UnityEditor.TestRunner, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
+        <HintPath>libs\UnityEditor.TestRunner.dll</HintPath>
+        <Private>false</Private>
+      </Reference>
+      <Reference Include="UnityEngine.TestRunner, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
+        <HintPath>libs\UnityEngine.TestRunner.dll</HintPath>
```

Comment:
Can't we use UPM to "restore" the package?

Or use this instead? https://www.nuget.org/packages/Troikatorz.UnityEngine.TestRunner/

Reply:
Yes, I saw this package, but I'm not sure we want to use it.

* Consistency - there is also a `UnityEditor.TestRunner` which is not on `nuget` currently.
* As far as I can tell, the given `nuget`  is just "there" and "not supported" from outside. `Unity` somehow changes\update `UnityEngine.TestRunner.dll`. We'll be adjusting to it, can't rely on the provided  `nuget` package.

One possible solution I can see for now is to "publish" both of the packages into `nuget` by ourselves. Otherwise, place them in `libs` and update when needed.

What do you think?